consent-ui/package.json

```diff
@@ -9,7 +9,7 @@
     "start": "ng serve",
     "serve:local": "ng serve --proxy-config=proxy-conf-local-backend.json",
     "serve:dev": "ng serve --proxy-config=proxy-conf-dev-backend.js",
-    "serve:docker": "ng serve --port=14200 --proxy-config=proxy-conf-docker-backend.json",
+    "serve:docker": "ng serve --port=4200 --proxy-config=proxy-conf-docker-backend.json",
```

Comment:
This is default port value - 4200, maybe this is accidential change?

Reply:
reverted